.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve your Python experience in Visual Studio
+title: ''
+labels: needs triage
+assignees: ''
+
+---
+
+**Describe the bug**
+[A clear and concise description of the bug]
+
+**Steps to Reproduce**
+1.
+2.
+3.
+
+**Expected behavior**
+[A clear and concise description of what you expected to happen]
+
+**Screenshots**
+[If applicable, add screenshots to help explain your problem]
+
+**Additional context**
+[Add any other information such as error message or stack trace]
```

Comment:
It's good to remind them to include screenshots, but I don't want them to feel like they shouldn't put them in the description or steps.

It's a lot of sections that they potentially have to delete when filing a bug.

Maybe change "Additional context" to "Additional context and screenshots"
Add any other information such as error message, stack trace or screenshots.

That would reduce it by one section.